ntoskrnl/io/iomgr/driver.c

```diff
@@ -5,7 +5,7 @@
  * PURPOSE:         Driver Object Management
  * PROGRAMMERS:     Alex Ionescu (alex.ionescu@reactos.org)
  *                  Filip Navara (navaraf@reactos.org)
- *                  Herv� Poussineau (hpoussin@reactos.org)
+ *                  Hervé Poussineau (hpoussin@reactos.org)
```

Comment:
Maybe this is broken now? (or it was broken before, github won't tell me)

Reply:
It was before :)